rust_sodium-sys/build.rs

```diff
@@ -118,14 +119,14 @@ fn main() {
 
     // Get path to gcc in order to guess location of libpthread.a
     let mut lib_search_dirs = vec![Path::new(&install_dir).join("lib")];
-    let where_cmd = Command::new("where");
+    let mut where_cmd = Command::new("where");
     let where_output = where_cmd.arg(gcc::Config::new().get_compiler().path())
         .output()
         .unwrap_or_else(|error| {
             panic!("Failed to run where command: {}", error);
         });
     if !where_output.status.success() {
-        panic!("\n{:?}\n{}\n{}\n",
+        panic!("\n{:?}\n{}\n",
```

Comment:
This should probably print `where_cmd`. It looks like the `{:?}` was added, but the `where_cmd` wasn't.